### ISSUE TITLE
GH Action: Pattern does not match any files

### ISSUE THREAD

**Issue by KuhlTime** [campbell-river-fire-department/electron-vue-boilerplate#8]
When creating a new release, with the `build-release.yml` action, the files can not be found. This is most likely because every file path has to be on a new line in order for the `softprops/action-gh-release` action to find those files.

```
Pattern 'electron-vite-fusion_0.1.0_linux_amd64.deb electron-vite-fusion_0.1.0_linux_arm64.deb electron-vite-fusion_0.1.0_linux_armv7l.deb electron-vite-fusion_0.1.0_mac_arm64.dmg electron-vite-fusion_0.1.0_mac_x64.dmg electron-vite-fusion_0.1.0_win_x64.exe' does not match any files.
```

**Comment by KuhlTime** [comment]
Output of the `jobs.list.outputs.files` variable: 
```
electron-vite-fusion_0.1.0_linux_amd64.deb electron-vite-fusion_0.1.0_linux_arm64.deb electron-vite-fusion_0.1.0_linux_armv7l.deb electron-vite-fusion_0.1.0_mac_arm64.dmg electron-vite-fusion_0.1.0_mac_x64.dmg electron-vite-fusion_0.1.0_win_x64.exe
```

**Comment by KuhlTime** [comment]
Created an adaptation of the [original action](https://github.com/softprops/action-gh-release) and changed the file input splitting logic from detecting a **newline-character** to detecting a **space-character**.

https://github.com/KuhlTime/action-gh-release/blob/7d997f4f17c96f9ea7cc59a5359336975d0f993a/src/util.ts#L43-L45

**Comment by KuhlTime** [comment]
FYI: https://togithub.com/softprops/action-gh-release/issues/79#issuecomment-1028928147

**Comment by KuhlTime** [comment]
Closed with e4d44e850393381a74c4cafd181009de7894c0cf
